### PR TITLE
BBL-19 | user-guide/identities completed

### PR DESCRIPTION
**Pull request by exequielrafaela**
### Commits on Jul 31, 2020
- @exequielrafaela - BBL-19 | user-guide/identities completed - 769e9d1
- @exequielrafaela  - BBL-19 small sintaxt improvement - 0c47732
- @exequielrafaela - BBL-19 minor adjustments to user-guide identities files - b33a9c5

### Extras
- Updating pre-req deps
- Adding medium to social nets.

### CODE REVIEW

File: docs/user-guide/identities/gpg.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+to manage AWS IAM Users credentials with **encryption to grant strong security**. 
+
+This **module** outputs commands and PGP messages which can be decrypted either using command line to get AWS Web Connsole
+user's password and user's secret key.   
+
+!!! warning "Notes for keybase users"
+    If possible, always use PGP encryption to prevent Terraform from keeping unencrypted password and access secret key 
+    in state file.
+
+!!! check "Keybase pre-requisits"
```
Comment (diego-ojeda-binbash) [inline docs/user-guide/identities/gpg.md:16]: Here it should be "pre-requisites" 😉

File: docs/user-guide/identities/gpg.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    gSLNsdpqP9lYaueFdLgAeR6a+EjvqZfq0hZAgoiymsb4Qtn4A7gmeGmNeDE4td1mVfgzuTZ9zhnSbAYlXNIiM4b0MeX4HrjFkT/Aq+A/rvgBeKhszWD4Ibh\
+    A4PgC+QPiJRb5kQ/mX8DheQfAHJ24iUZk1jh6AsA" | base64 --decode > encrypted_pass
+    ```
+    3. Run `gpg --decrypt a_file_with_your_pass` (in the path you've executed 2.) to effectively decrypt your pass using
+     your gpg key and its passphrase
+    ```bash
+    $ gpg --decrypt encrypted_pass
+    
+    You need a passphrase to unlock the secret key for
+    user: "Exequiel Barrirero (Exequiel Barrirero AWS org binbash-user acct gpg key w/ passphrase) <exequiel.barrirero@binbash.com.ar>"
```
Comment (diego-ojeda-binbash) [inline docs/user-guide/identities/gpg.md:55]: @exequielrafaela Would you say that it's prudent to publicly expose your email (or any BB member for that matter)?

File: docs/user-guide/identities/gpg.md (hunk 1 above)
Comment (exequielrafaela) [reply docs/user-guide/identities/gpg.md]: @diego-ojeda-binbash OJ I've just correted this :) Thanks!

File: docs/user-guide/identities/gpg.md (hunk 2 above)
Comment (exequielrafaela) [reply docs/user-guide/identities/gpg.md]: @diego-ojeda-binbash OJ my bad, fixed too. Thanks for your review here!

**Pull request merged**